cmd/server/config.go

```diff
@@ -11,9 +10,12 @@ import (
 )
 
 const (
-	ocidRoot            = "/var/lib/ocid"
+	ocidRoot            = "/var/lib/containers"
+	ocidRunRoot         = "/var/run/containers"
 	conmonPath          = "/usr/libexec/ocid/conmon"
-	pausePath           = "/usr/libexec/ocid/pause"
+	pauseImage          = "kubernetes/pause"
```

Comment:
right, I believe we should have still a `pausePath` flag for now in order to setup pause directly w/o pulling anything from the net. @cyphar does that sound good? we'll have either the option to pull the image or run with whatever `pausePath` an admin provides